client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -22,20 +24,37 @@ const DesignSelector: FunctionComponent< Props > = () => {
 	const { selectedDesign, siteVertical } = useSelect( select =>
 		select( ONBOARD_STORE ).getState()
 	);
-	const { setSelectedDesign } = useDispatch( ONBOARD_STORE );
+	const { setSelectedDesign, resetOnboardStore } = useDispatch( ONBOARD_STORE );
+	const makePath = usePath();
+
+	const handleStartOverButtonClick = () => {
+		resetOnboardStore();
+	};
 
 	return (
 		<div
 			className="design-selector gutenboarding-color-coded"
 			data-vertical={ siteVertical?.label }
 		>
 			<div className="design-selector__header">
-				<h1 className="design-selector__title">{ NO__( 'Choose a starting design' ) }</h1>
-				<h2 className="design-selector__subtitle">
-					{ NO__(
-						'Get started with one of our top website layouts. You can always change it later'
-					) }
-				</h2>
+				<div className="design-selector__header-section design-selector__header-section--title">
+					<h1 className="design-selector__title">{ NO__( 'Choose a starting design' ) }</h1>
+					<h2 className="design-selector__subtitle">
+						{ NO__(
+							'Get started with one of our top website layouts. You can always change it later'
+						) }
+					</h2>
+				</div>
+				<div className="design-selector__header-section">
+					<Link
+						className="design-selector__start-over-button"
+						onClick={ handleStartOverButtonClick }
+						to={ makePath( Step.IntentGathering ) }
+						isLink
```

Comment:
Heh, `<Link isLink />` looks little weird, but let's not worry about it now. 😁

Reply:
Yes, actually, without `isLink`, additional overrides for `:hover` is needed (which draws a border around the button), the specificity for the`:hover` state in the `Button` component is too high, so I wanted to avoid that.  I'm actually okay with showing a border around the button when hovered, but design might think otherwise.  It's up for interpretation.

Reply:
These are props from `@wordpress/components` `Button`. It does look a bit weird, but not something we should get  hung up on.